server/src/main/java/com/lunchtime/models/Order.java

```diff
@@ -0,0 +1,83 @@
+package com.lunchtime.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.ColumnDefault;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.time.Instant;
+import java.util.Date;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+@Data
+@Table(name = "ordering")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "person_id", referencedColumnName = "id")
+    private Person person;
+
+    @Column(name = "waiter_id")
+    private Long waiter;
+
+    @NotNull
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "start_time")
+    private Date startTime;
+
+    @NotNull
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "finish_time")
+    private Date finishTime;
+
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "status_id", referencedColumnName = "id")
+    private OrderStatus status;
+
+    @Size(max = 999)
+    @Column(name = "description")
+    private String description;
+
+    @NotNull
+    @Column(name = "visitors")
+    private Integer visitors;
+
+    @NotNull
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "table_id", referencedColumnName = "id")
+    private RestaurantTable table;
+
+    @Column(name = "dish_details_id")
+    private Long dishDetails;
```

Comment:
Can't find out what should be `dishDetails`. It should be dish/dishes that user ordered (or not ordered) while booked table, shouldn't it?

Reply:
Yes, may be)

Reply:
So I guess this may be temporarily decision to persist them in `Long` type?

Reply:
Yes

Reply:
I think it is better to write this near `Long` usage so that other developers could understand it.

Reply:
What?

Reply:
Something like comments, or documentation.

Reply:
I don't write useless comments. If developer want to know what is it, he or she can open database scheme

![image](https://user-images.githubusercontent.com/6152066/79738333-468de180-8305-11ea-87eb-c9bab14539eb.png)

Reply:
So, you don't use useless comments but create useless actions for another developer? You told this is temporary desicison to use `Long` here.